lib/pages/pool/AddLiquidity/AddLiquidity.dart

```diff
@@ -310,6 +312,7 @@ class _AddLiquidityState extends State<AddLiquidity> {
                               BoxConstraints(maxWidth: tokenContainerWdt * 0.5),
                           child: IntrinsicWidth(
                             child: TextFormField(
+                              readOnly: ((tknNum == 2) & (state.status == BlocStatus.success)) ? isReadOnly : !isReadOnly,
```

Comment:
Why are we using bit manipulation here? Shouldn't it be `readOnly: ((tknNum == 2) && (state.status == BlocStatus.success)) ? isReadOnly : !isReadOnly,`